app/models/miq_region.rb

```diff
@@ -13,9 +13,9 @@ class MiqRegion < ApplicationRecord
   virtual_has_many :miq_servers,            :class_name => "MiqServer"
   virtual_has_many :active_miq_servers,     :class_name => "MiqServer"
 
-  virtual_has_many :vms_and_templates,      :uses => :all_relationships
-  virtual_has_many :miq_templates,          :uses => :all_relationships
-  virtual_has_many :vms,                    :uses => :all_relationships
+  virtual_has_many :vms_and_templates
+  virtual_has_many :miq_templates
+  virtual_has_many :vms
```

Comment:
Here is the code I read:

```ruby
    alias_method :all_vms,                :vms
   def vms
     Vm.in_region(region_number)
   end
```

So I interpret that it is a query with VMs and uses no relationships.

Think I'm missing what you said.